packages/firebase_ui_auth/pubspec.yaml

```diff
@@ -28,6 +28,10 @@ dev_dependencies:
   flutter_lints: ^2.0.0
   mockito: ^5.2.0
 
+dependency_overrides:
+  firebase_ui_oauth:
+    path: ../firebase_ui_oauth
```

Comment:
We should remove this. Melos handles this for us

Reply:
Yes it should be removed before release. I was just doing what I do for my PRs on the official flutter plugins repo.

Reply:
I removed it